pkg/remoting/getty/listener.go

```diff
@@ -113,8 +113,6 @@ func (g *gettyClientHandler) OnMessage(session getty.Session, pkg interface{}) {
 }
 
 func (g *gettyClientHandler) OnCron(session getty.Session) {
-	log.Debug("session{%s} Oncron executing", session.Stat())
-	g.transferBeatHeart(session, message.HeartBeatMessagePing)
```

Comment:
why remove heartbeat?

Reply:
I will sort out the corresponding timing and make further adjustments

Reply:
> I hope to delay the interval between transaction submissions by means of timers in the code. The heartbeat may cause getty to disconnect and reconnect, and the time is uncontrollable.

If the heartbeat is closed, it may cause getty to disconnect and reconnect

Reply:
> We should register the branch transaction in listener#onOpen before the session is reopened

you mean register resource on Open?

Reply:
Yes, I want to call GetResourceManager().branchRegister before registering the resource

Reply:
> Yes, I want to call GetResourceManager().branchRegister before registering the resource

I don't know if I understand it right or not, in my understanding, branch transaction is regietered  when a branch rpc is called, so what are you trying to say here is register  resource when it's Open ?